src/common/components/navigation/MenuItem.tsx

```diff
@@ -19,26 +13,46 @@ export type MenuItemProps = {
   isFallback?: boolean;
   isDisabled?: boolean;
   tooltipTextRows?: string[];
+  isBeforeSelected?: boolean;
+  isAfterSelected?: boolean;
+  selected?: boolean;
 };
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    disabled: {
-      color: theme.palette.text.disabled,
+const useStyles = makeStyles((theme) => ({
+  listItem: (props: MenuItemProps) => ({
+    height: "50px",
+    color: props.isDisabled
+      ? theme.palette.text.disabled
+      : props.selected
+      ? "#d7d9e2"
+      : "#9a9ca4",
```

Comment:
To ease the readability, maybe let's move the logic to a separate function that can be called here in case there's 2-3 ternary expressions for one property?